executor.go

```diff
@@ -512,7 +518,9 @@ func resolveField(eCtx *ExecutionContext, parentType *Object, source interface{}
 	var returnType Output
 	defer func() (interface{}, resolveFieldResultState) {
 		if r := recover(); r != nil {
-
+			if eCtx.PanicHandler != nil {
+				eCtx.PanicHandler(eCtx.Context, r)
```

Comment:
It will capture all erros from GraphQL. If you see [here](https://github.com/graphql-go/graphql/blob/451533e82374cae4032ce83553bb374af94c5a62/executor.go#L536). All errors was given by panic(). Changing the `return` to value (`error`), can solve the problem.